pkg/git/gogit_test.go

```diff
@@ -84,6 +89,31 @@ var _ = Describe("Clone", func() {
 	})
 })
 
+var _ = Describe("ValidateAccess", func() {
+	It("validate access to a given repository successfully", func() {
+		err := gitClient.ValidateAccess(context.Background(), "https://github.com/githubtraining/hellogitworld", "master")
+		Expect(err).ShouldNot(HaveOccurred())
+	})
+
+	It("should not fail on an empty repo", func() {
+
+		fakeGoGit := &gitfakes.FakeGoGitI{}
+
+		fakeGoGit.PlainCloneContextReturns(nil, transport.ErrEmptyRemoteRepository)
+
+		fakeGitClient := git.New(nil, fakeGoGit)
+
+		err := fakeGitClient.ValidateAccess(context.Background(), "https://github.com/githubtraining/hellogitworld", "master")
+
+		Expect(err).ShouldNot(HaveOccurred())
```

Comment:
We should really have more concrete assertions on the errors.  Which error occurred?  Check for that instead of the fact that it just occurred.

Reply:
Nevermind, I read this wrong.